app/src/main/java/com/asfoundation/wallet/billing/partners/OemIdExtractorV2.kt

```diff
@@ -1,12 +1,15 @@
 package com.asfoundation.wallet.billing.partners
 
 import android.content.Context
-import com.aptoide.injectorextractor.Extractor
+import com.aptoide.apk.injector.extractor.data.Extractor
+import com.aptoide.apk.injector.extractor.data.ExtractorV1
+import com.aptoide.apk.injector.extractor.data.ExtractorV2
 import io.reactivex.Single
 import java.io.File
 
 class OemIdExtractorV2 @JvmOverloads constructor(private val context: Context,
-                                                 private val extractor: Extractor = Extractor()) :
+                                                 private val extractor: Extractor = Extractor(
+                                                     ExtractorV1(), ExtractorV2())) :
```

Comment:
Discuss: Shouldn't this Extractor be provided instead of being created here

Reply:
Extractor is a final class. As such there's no gain in exposing it I believe.